perception/navigator_vision/nodes/pinger_finder.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
```

Comment:
David said this isn't ready to merge. Probably shouldn't add this and let him do it or talk with him.